types/file-manager/index.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for file-manager 2.0
+// Project: https://github.com/DoctorMcKay/node-file-manager#readme
+// Definitions by: repistory <https://github.com/repository>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import EventEmitter = require('events');
+
+export = FileManager;
+
+declare class FileManager extends EventEmitter {
+    /**
```

Comment:
I think you are missing the `directory` property, ([see here](https://github.com/DoctorMcKay/node-file-manager#directory)), which can be changed and assigned to by the user.

Reply:
oops, added that now!